src/osbuild-manifests/aws_tag.py

```diff
@@ -0,0 +1,90 @@
+import subprocess
+import json
+import argparse
+ 
+def main():
+    parser = argparse.ArgumentParser()
+    parser.add_argument('--stream', dest='stream', type=str, help='Fedora stream', required=True)
+    parser.add_argument('--dry-run', dest='dry_run', help='Check if the resources have tags but not add them', action='store_true')
+    args = parser.parse_args()
+ 
+    builds = getBuildsForStream(args.stream)
+    for build in builds:
+        build_id=build['id']
+        arches=build['arches']
+        for arch in arches:
+            print(f"Parsing AMIs for {build_id} for {arch}")
+            buildFetch(args.stream, build_id, arch)
+            meta = open(f'builds/{build_id}/{arch}/meta.json')
+            data = json.load(meta)
+ 
+            if 'amis' in data.keys():
+                amis = data['amis']
+            else:
+                print(f"{build_id} does not have any AMIs for {arch} in meta.json")
+                break

```

Comment:
I think we need to continue and not break here? if we break then we won't check all architectures for a build ID

```suggestion
                continue

```